src/cdk/private/visually-hidden/visually-hidden.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ChangeDetectionStrategy, Component, ViewEncapsulation} from '@angular/core';
+
+/**
+ * Component used to load the .cdk-visually-hidden styles.
+ * @docs-private
+ */
+@Component({
+  standalone: true,
+  styleUrl: 'visually-hidden.css',
+  exportAs: 'cdkVisuallyHidden',
```

Comment:
We can drop the `exportAs`.

Reply:
I had to add this in because if I drop the `exportAs` I get a duplicate component ID error :( Is there another way to resolve this error?

Reply:
The full error is:
```
Detected identical metadata between following components:
- _MatBadgeStyleLoader and _VisuallyHiddenLoader

The metadata of one of each of these components should be changed to be slightly different in order to avoid conflicts at runtime.
```

Reply:
I see, then it should be fine. Another way to work around it is to add a host binding.